build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
             }
             credentials {
                 username "mapbox"
-                password = project.hasProperty('SDK_REGISTRY_TOKEN') ? project.property('SDK_REGISTRY_TOKEN') : System.getenv('SDK_REGISTRY_TOKEN')
+                password "sk.eyJ1IjoidGFyaWdvLW1hcGJveCIsImEiOiJja2J1bGhwdG4wNGwxMnVwYXJvcGJqczByIn0.R0qYY8UsebRKMaDqTn0xbQ"
```

Comment:
shouldn't commit the token to the project.

Reply:
:+1:

Reply:
going to rotate the token